docs/swagger.yaml

```diff
@@ -204,6 +204,44 @@ paths:
                         type: string
                         description: message
                         example: Password reset email sent
+  /auth/password/{token}:
```

Comment:
I think there are examples in the current docs we have. The `token` should be defined as a parameter. 

Currently, the docs are not showing anything under `Parameters` for this endpoint

Reply:
done @jocrah